.github/workflows/release.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - run: echo "Using VehicleApp version ${{ steps.get_version.outputs.version-without-v }} from tag"
 
       - name: Login to GitHub Container Registry
```

Comment:
please check this workflow again there are some mismatches between waiting and downloading. only wait for and download whats really needed